packages/components/src/components/input/input.ts

```diff
@@ -642,28 +643,20 @@ export default class SdInput extends SolidElement implements SolidFormControl {
                   <button
                     aria-label=${this.localize.term(this.passwordVisible ? 'hidePassword' : 'showPassword')}
                     part="password-toggle-button"
-                    class="flex items-center"
+                    class=${cx('flex items-center sd-interactive', iconMarginLeft)}
                     type="button"
                     @click=${this.handlePasswordToggle}
-                    tabindex="-1"
+                    aria-pressed=${this.passwordVisible ? 'true' : 'false'}
```

Comment:
Does this work together with the current label? My expectation would be that then the label would have to stay the same, as now the following would happen when the password is visible:

- `label="hide password"` 
- `aria-pressed="true"`

Which doesn't really make sense, right?

(I know, that this propsal came from @matuzo, but I believe this combination doesn't make sense. So either we get rid of "aria-pressed" or we convert the label to being "non-ternary" and getting rid of "hidePassword".

Reply:
Retested again and definetely does not make sense here, removed the aria attribute and kept the label which feels more informative IMO